ogr/services/pagure.py

```diff
@@ -512,6 +549,45 @@ def get_git_urls(self) -> Dict[str, str]:
         return_value = self._call_project_api("git", "urls")
         return return_value["urls"]
 
+    def _issue_from_pagure_dict(self, issue_dict: dict) -> Issue:
+        if self.namespace is None:
+            url = "/".join(
+                [
+                    self.service.instance_url,
+                    self.repo,
+                    "issue",
+                    str(issue_dict["id"]),
+                ]
+            )
+        else:
+            url = "/".join(
+                [
+                    self.service.instance_url,
+                    self.namespace,
+                    self.repo,
+                    "issue",
+                    str(issue_dict["id"]),
+                ]
+            )
```

Comment:
Try using [self.getproject_url](https://github.com/packit-service/ogr/blob/0d60bcc68e53db7d75cb898c036c7b4c529f045a/ogr/services/pagure.py#L296)...

This should work for you:
```python
url = self.get_project_url("issue", str(issue_dict["id"]))
```